src/net.cpp

```diff
@@ -1347,8 +1347,6 @@ void CConnman::SocketHandler()
 
     int nSelect = select(have_fds ? hSocketMax + 1 : 0,
                          &fdsetRecv, &fdsetSend, &fdsetError, &timeout);
-    if (interruptNet)
```

Comment:
~~Move this into `SocketHandler` after the `SocketEvents` call? Or this method cool return a bool, true if it completes, false if it is interrupted.~~

NOTE: this was fixed in the commit `maybe fix things?`